backend/dal/Migrations/Initial/Up/PostUp/01-AdministrativeAreas.sql

```diff
@@ -0,0 +1,352 @@
+PRINT N'Adding [PIMS_ADMINISTRATIVE_AREA]'
+
+MERGE INTO dbo.[PIMS_ADMINISTRATIVE_AREA] dest
+USING ( VALUES
+('100 Mile House','2','Cariboo Regional District'),
+('150 Mile House','3',NULL),
+('Abbotsford','4','Fraser Valley Regional District'),
+('Agassiz','5',NULL),
+('Ahousaht','6',NULL),
+('Albert Canyon','7',NULL),
+('Aldergrove','8',NULL),
+('Alert Bay','9','Regional District of Mount Waddington'),
+('Alexis Creek','10',NULL),
+('Aleza Lake','11',NULL),
+('Allison Pass','12',NULL),
+('Anahim Lake','13',NULL),
+('Anmore','Anmore','Metro Vancouver Regional District'),
```

Comment:
seems odd that some of the abbreviations are just the name again, especially when the name in question is kinda long.

Reply:
I didn't change this, it came from the original PIMS project.  I suspect the data isn't perfect.